routing/pathfind.go

```diff
@@ -589,6 +589,13 @@ func findPath(g *graphParams, r *restrictParams,
 			return
 		}
 
+		// If this edge was constructed from a hop hint, we won't have access to
+		// its max HTLC. Therefore, only consider discarding this edge here if
+		// the field is set.
+		if edge.MaxHTLC != 0 && edge.MaxHTLC < amountToSend {
```

Comment:
👍

Reply:
I moved the routing to a separate PR as well: #2437